resources/css/app.css

```
@@ -58,6 +58,135 @@
+    opacity: 0.6;
+    transition: opacity 150ms cubic-bezier(0.4, 0, 0.2, 1);
+}
+
+.group:hover .rfa-lazy-actions { opacity: 1; }
+.rfa-lazy-comments { gap: 0.125rem; }
+
+.rfa-lazy-button {
+    width: 2rem;
+    height: 2rem;
+    flex: none;
+    background-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)